app/config.py.example

```diff
@@ -35,6 +35,8 @@ WEBSITE_LANGUAGE = "en"
 # Get a free API KEY for Astronomy feature @ www.weatherapi.com/signup.aspx
 ASTRONOMY_API_KEY = os.getenv('ASTRONOMY_API_KEY')
 WEATHER_API_KEY = os.getenv('WEATHER_API_KEY')
+CLIENT_SECRET_FILE = os.environ.get('CLIENT_SECRET')
```

Comment:
Add documentation about what this variable means

Reply:
like a link?

Reply:
Done.